src/shared/components/query/QueryStore.ts

```diff
@@ -192,61 +174,16 @@ export class QueryStore
 		return Object.keys(ret);
 	}
 
-	readonly temporaryVirtualCohortId = remoteData({

```

Comment:
Could you give me some info how the temporaryVirtualCohort works in the new implementation?

Reply:
`temporaryVirtualCohort` was legacy and bit hard to understand (btw that method is not invoked at all). So just moved the api call and object mapping to `src/shared/api/sessionServiceAPI.ts`